projects/reservoir-protocol/index.js

```diff
@@ -95,9 +95,15 @@ Object.keys(config).forEach(chain => {
       tvl: async (api) => {
 
         let balance = await api.call({ abi: 'function balanceOf(address) view returns (uint256)', target: '0x7519403E12111ff6b710877Fcd821D0c12CAF43A', params: ['0x9A319b57B80c50f8B19DB35D3224655F3aDd8E4f'] })
+        let value = await api.call({ abi: 'function balanceOf(address) view returns (uint256)', target: '0x7519403E12111ff6b710877Fcd821D0c12CAF43A', params: ['0x9A319b57B80c50f8B19DB35D3224655F3aDd8E4f'] })
 
         api.add('0x7519403E12111ff6b710877Fcd821D0c12CAF43A', balance)
 
+        balance = await api.call({ abi: 'function balanceOf(address) view returns (uint256)', target: '0xd8f824d4252caE7d5E49B95d47B0EfAfe6f2d570', params: ['0x9A319b57B80c50f8B19DB35D3224655F3aDd8E4f'] })
+        value = await api.call({ abi: 'function convertToAssets(uint256) view returns (uint256)', target: '0xd8f824d4252caE7d5E49B95d47B0EfAfe6f2d570', params: [balance] })
+
+        api.add(ADDRESSES.ethereum.USDC, value)
```

Comment:
please export a balance of the vault contract at 0xd8f824d4252caE7d5E49B95d47B0EfAfe6f2d570 rather than the underlying